README.md

```diff
@@ -48,7 +48,7 @@ Choose one of the following options:
   * A custom build of [`Modernizr`](https://modernizr.com/) for feature
     detection
   * [`Apache Server Configs`](https://github.com/h5bp/server-configs-apache)
-    that, among other, improve the web site's performance and security
+    that, among others, improve the web site's performance and security
```

Comment:
```suggestion
    that improve the web site's performance and security
```

Can you make the above change? I'd prefer for you get the commit credit for pointing out the typo, but I'd like to simplify the sentence at the same time